indico/modules/rb/client/js/common/bookings/BookingExportModal.jsx

```diff
@@ -0,0 +1,130 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import React from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Form as FinalForm} from 'react-final-form';
+import {Button, Form, Modal} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import {Translate} from 'indico/react/i18n';
+import {FinalDatePeriod} from 'indico/react/components';
+import {FinalRadio} from 'indico/react/forms';
+import {actions as bookingActions} from '../bookings';
+import FinalRoomSelector from '../../components/RoomSelector';
+
+function validate({format, dates, rooms}) {
+  const errors = {};
+  if (!format) {
+    errors.type = Translate.string('Please choose a file format to export the bookings.');
+  }
+  if (!dates || !Object.values(dates).every(x => x)) {
+    errors.dates = Translate.string('Please choose a valid period.');
+  }
+  if (!rooms || !rooms.length) {
+    errors.rooms = Translate.string('Please choose at least one room to export its bookings.');
+  }
+  return errors;
+}
+
+const BookingExportModal = props => {
```

Comment:
I also didn't do it a few times but I think we should try to use `function` statements for top-level components, especially those that are exported. That way it's guaranteed that the component name is properly set in all cases.